src/main/java/discord/TwoDee.java

```diff
@@ -32,7 +28,7 @@ public static void main(String[] args) {
             new DiscordApiBuilder().setToken(token).setAllIntentsExcept(Intent.GUILD_PRESENCES).setUserCacheEnabled(true).addListener(velen).login().thenAccept(api -> {
                         // Uncomment this line when a command is altered
                         // TODO do this a smarter way
-                        velen.registerAllSlashCommands(api);
+                        // velen.registerAllSlashCommands(api);
```

Comment:
I once again want to point out, THE ONLY REASON YOU CAN COMMENT THIS OUT IS BECAUSE YOU ARE USING THE MAIN DISCORD FOR TESTING. This would cause any new commands I add, to not work. So stop commenting it out.

Reply:
Uncommented